openpype/hosts/fusion/plugins/publish/increment_current_file.py

```diff
@@ -1,29 +1,38 @@
 import pyblish.api
 
+from openpype.pipeline import OptionalPyblishPluginMixin
 
-class FusionIncrementCurrentFile(pyblish.api.ContextPlugin):
+
+class FusionIncrementCurrentFile(
+    pyblish.api.ContextPlugin, OptionalPyblishPluginMixin
+):
     """Increment the current file.
 
     Saves the current file with an increased version number.
 
     """
 
-    label = "Increment current file"
+    label = "Increment workfile version"
     order = pyblish.api.IntegratorOrder + 9.0
     hosts = ["fusion"]
-    families = ["workfile"]
     optional = True
 
     def process(self, context):
+        if not self.is_active(context.data):
+            return
 
         from openpype.lib import version_up
         from openpype.pipeline.publish import get_errored_plugins_from_context
 
         errored_plugins = get_errored_plugins_from_context(context)
-        if any(plugin.__name__ == "FusionSubmitDeadline"
-                for plugin in errored_plugins):
-            raise RuntimeError("Skipping incrementing current file because "
-                               "submission to render farm failed.")
+        if any(
+            plugin.__name__ == "FusionSubmitDeadline"
+            for plugin in errored_plugins
+        ):
+            raise RuntimeError(
+                "Skipping incrementing current file because "
+                "submission to render farm failed."
+            )
```

Comment:
Just two things - raising Exception here will stop the publish, so maybe "Skipping ..." isn't the right word here as (at least for me) it suggest it will continue, just with the warning. I know that this is at the Integration phase so all integrators before already did their job, but still - if we are rising error, maybe the message should be more imperative 🤔 

Also, it should be `KnownPublishError` probably (`from openpype.pipeline import KnownPublishError`)

Reply:
Thanks for the note :) Updated to this!